packages/amplify-cli-core/src/help/commands-info.ts

```diff
@@ -59,7 +59,12 @@ export const commandsInfo: Array<CommandInfo> = [
     command: 'configure',
     commandDescription: 'Configure the CLI to work with your AWS profile',
     commandUsage: 'amplify configure [subcommand]',
-    commandFlags: [],
+    commandFlags: [
+      { short: '', long: 'usage-data-on', flagDescription: 'Turn on usage data sharing.' },
+      { short: '', long: 'usage-data-off', flagDescription: 'Turn off usage data sharing.' },
+      { short: '', long: 'share-project-config-on', flagDescription: 'Turn on non-sensitive configurations sharing on failures.' },
```

Comment:
```suggestion
      { short: '', long: 'share-project-config-on', flagDescription: 'Turn on non-sensitive project configuration sharing on failures.' },
```

Reply:
adding "project configuration" to help bridge the understanding with its flag "share-project-config-on/off"

Reply:
Cool. I'll do a global search to update it everywhere.